### PR TITLE
Revise introduction to StatefulSet tutorial

### PR DESCRIPTION
**Pull request by sftim**
Split out from https://github.com/kubernetes/website/pull/42767 to help reviewers. [preview](https://deploy-preview-43133--kubernetes-io-main-staging.netlify.app/docs/tutorials/stateful-application/basic-stateful-set/).

### CODE REVIEW

File: content/en/docs/tutorials/stateful-application/basic-stateful-set.md [hunk 1]
```diff
@@ -27,14 +27,25 @@ following Kubernetes concepts:
...
+If you are using an existing cluster, make sure that it's OK to use that
+cluster's default namespace to practice. Ideally, practice in a cluster
+that doesn't run any real workloads.
+
+It's also useful to read the concept page about [StatefulSets](/docs/concepts/workloads/controllers/statefulset/).
+
 {{< note >}}
 This tutorial assumes that your cluster is configured to dynamically provision
-PersistentVolumes. If your cluster is not configured to do so, you
+PersistentVolumes (you'll also need to have a [default StorageClass](/docs/concepts/storage/storage-classes/#default-storageclass)).
```
Comment (tengqm) [inline content/en/docs/tutorials/stateful-application/basic-stateful-set.md:43]: Maybe we can drop the parenthesis.

Comment (sftim) [reply content/en/docs/tutorials/stateful-application/basic-stateful-set.md]: Done.

**Comment by windsonsea** [comment]
The [preview](https://deploy-preview-43133--kubernetes-io-main-staging.netlify.app/docs/tutorials/stateful-application/basic-stateful-set/#before-you-begin) looks good.
/lgtm

**Review by Affan-7 — APPROVED** [review]

**Pull request merged**